pusher/pusher_client.py

```diff
@@ -41,12 +41,26 @@ def __init__(
             timeout=5, cluster=None,
             encryption_master_key=None,
             json_encoder=None, json_decoder=None,
-            backend=None, **backend_options):
+            backend=None, max_num_channels = None,
```

Comment:
Please keep formatting consistent with the rest of the file:

```suggestion
            backend=None, max_num_channels=None,
```